packages/matchbox/src/components/Expandable/Expandable.js

```diff
@@ -84,6 +85,7 @@ function Expandable(props) {
         aria-hidden={!isOpen}
         className={contentClasses}
         id={id}
+        data-id="expandable-content"
```

Comment:
Looks like you'll need to update the unit test snapshots for these new props, lemme know if you'd like me to run you through that